### ISSUE TITLE
Sulphurpool theme for Sublime Text is not available

### ISSUE THREAD

**Issue by ink-splatters** [atelierbram/syntax-highlighting#5]
Hello,

this links are unavailable anymore:

https://atelierbram.github.io/syntax-highlighting/atelier-schemes/output/textmate/textmate.zip
https://github.com/atelierbram/syntax-highlighting/tree/master/atelier-schemes/output/textmate

Can you please explain why.. The theme is really cool, I use it for iTerm2 and would like to use for Sublime Text..

Thanks!

Best regards,
Peter

**Comment by atelierbram** [comment]
Hi Peter, 

Thanks for the heads up! After a redesign of this project I chose to archive the atelier-schemes output - but have them still available here -  since I don't think they will change much in the future. But obviously I forgot to change the links to the textmate files which live over here now: 
https://github.com/atelierbram/syntax-highlighting/tree/master/docs/archive/atelier-schemes/output/textmate
Fixed now, ... btw always nice to know these themes are being liked and used.
